src/pages/HorseRace/components/Back/Back_Logic.tsx

```diff
@@ -205,6 +208,16 @@ const BackLogic: React.FC<Props> = ({ runner }) => {
   );
   const [wagerAmount, setWagerAmount] = useState<number>(0);
   const [signature, setSignature] = useState<EcSignature>();
+  const config = useConfig();
+
+  const marketData = useMarketDetail(selectedMarketAddress);
+  const { data: balanceData } = useBalance({
+    address: address as Address,
+    token:
+      marketData?.name && marketData.name.includes("DAI")
+        ? (config?.tokenAddresses.DAI as Address)
```

Comment:
these can be imported from `StaticConfig`

Reply:
also is `as Address` necessary? is there not an `Address` constructor?

Reply:
1. no, what in staticConfig are mock asset address, what i got from dropdown is market contract address, 
so it has to be this way.
2. you are right.